ova/Libraries/elastic_functions.sh

```diff
@@ -88,12 +88,12 @@ configure_kibana(){
 
 install_kibana_app(){
 
-    if [ "${STATUS_PACKAGES}" = "stable" ]; then
+    if [ "${PACKAGES_REPOSITORY}" = "prod" ]; then
```

Comment:
If this PR goes to 4.x the URL won't match.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/462/commits/3e33cfd6fe2cfa31bc0e7627ba1db39146b5a746